README.md

```diff
@@ -70,13 +70,31 @@ This starts your app in development mode, rebuilding assets on file changes.
 
 ### Running Locally
 
-Running `npm run dev` will also trigger the Netlify local development environment which will pull in all the [environment variables](https://docs.netlify.com/configure-builds/environment-variables/#declare-variables) of your Netlify project. You can learn more about this project's Supabase environment variables in [the Database section below](#database). With Netlify dev you can also:
+The Remix dev server starts your app in development mode, rebuilding assets on file changes. To start the Remix dev server:
```

Comment:
This is pretty much the writing you had updated for the Remix Netlify functions template @KyleBlankRollins, but I retrofitted into what was here already.

Reply:
I just have one [dust] comment below. The rest looks great! Thanks, @nickytonline.